plugins/net_plugin/net_plugin.cpp

```diff
@@ -965,7 +965,7 @@ namespace eosio {
    }
 
    void connection::enqueue( const net_message &m, bool trigger_send ) {
-      bool close_after_send = false;
+      go_away_reason close_after_send = no_reason;
```

Comment:
`go_away_reason` defined as `enum` in protocol.hpp

Reply:
That's right. Semantically the "close_after_send" value's role is unchanged. Passing the actual reason value gives the added benefit of being able to report why a connection is being actively closed.